File: wamp/www/generate_id.php
<?php
if(!function_exists('uniqid'))
{
function uniqid() {
    # Cette fonction renvoie un identifiant unique
    $micro = substr (microtime(), 2, 6) ;
    $concat = time() . $micro ;
    $dec_1 = substr ($concat, 0, 8) ;
    $dec_2 = substr ($concat, 8, 8) ;
    $hex_1 = dechex ($dec_1) ;
    $hex_2 = dechex ($dec_2) ;
    $id = $hex_1 . $hex_2 ;
    return $id ;
}
}
echo uniqid();
?>

File: vps/generate_id.py
from time import time

def uniqid():
    t = str(int(time()*1000000))
    dec1 = int(t[0:8])
    dec2 = int(t[8:16])
    return '%x%x' % (dec1,dec2)

def main():
    print uniqid()

if __name__=='__main__':
    main()
